core/src/models/OAuthRequest.ts

```diff
@@ -64,6 +64,7 @@ export class OAuthRequest extends CommonModel<OAuthRequest> {
       type: this.type,
       consumed: this.consumed,
       provider: this.provider,
+      token: this.consumed ? null : this.token,
```

Comment:
This seems a bit dangerous to expose, but... I guess it's OK for now.  We could come up with another scheme to load values from `GROUPAROO_OPTION__OAUTH_{ID}` sort of like how we do with ENV, but we wait until later for that